Extensions/PanelSpriteObject/panelspriteruntimeobject-pixi-renderer.ts

```diff
@@ -57,34 +59,35 @@ namespace gdjs {
       for (let i = 0; i < this._borderSprites.length; ++i) {
         this._spritesContainer.addChild(this._borderSprites[i]);
       }
-      this._alpha = this._spritesContainer.alpha;
+      this._wrapperContainer.addChild(this._spritesContainer);
+      this._alpha = this._wrapperContainer.alpha;
       runtimeScene
         .getLayer('')
         .getRenderer()
-        .addRendererObject(this._spritesContainer, runtimeObject.getZOrder());
+        .addRendererObject(this._wrapperContainer, runtimeObject.getZOrder());
     }
 
     getRendererObject() {
-      return this._spritesContainer;
+      return this._wrapperContainer;
```

Comment:
This is correct 👍 The "top most" container is indeed the one considered as the "renderer object". It will be the one in the future to receive the Pixi filters (in the PR #2736 being worked on by @HarsimranVirk).